en/getting-started/first-game/index.md

```diff
@@ -276,72 +579,230 @@ After reading and understanding the capabilities of the __Animation Editor__ cha
     }
     ```
 
-    Click the __Play__ button. When playing, click the left and right mouse buttons, you can see the new jump effect in action:
+6. Click the **Preview** button to see the new jump effect in the open browse by clicking the left or right mouse button.
```

Comment:
Click the **Preview** button to see the new jump animation in the open browser by clicking the left or right mouse button.